src/browser/api/application.js

```diff
@@ -169,14 +170,15 @@ Application.create = function(opts, configUrl = '', parentIdentity = {}) {
     }
 
     const appObj = createAppObj(uuid, opts, configUrl);
+    const app = coreState.appByUuid(opts.uuid);
 
     if (parentIdentity && parentIdentity.uuid) {
         // This is a reference to the meta `app` object that is stored in core state,
         // not the actual `application` object created above. Here we are attaching the parent
         // identity to it.
-        const app = coreState.appByUuid(opts.uuid);
         app.parentUuid = parentUuid;
     }
+    app.initialAppOptions = initialAppOptions;
```

Comment:
By adding this, we should also update app's shape [here](https://github.com/tgoc99/core/blob/667ea92069e9cdb69de941ecf34edcb122a65083/src/shapes.ts#L76)

Reply:
Also, is this different from app's `_options` ?

Reply:
`_options` are mutated some instead of as-received. `name` is overwritten with `uuid`, `autoShow` is overwritten to false, etc.